bouncer/src/repo/manipulator/modelutility/repo_mesh_map_reorganiser.cpp

```diff
@@ -157,7 +157,14 @@ repo::core::model::MeshNode MeshMapReorganiser::getRemappedMesh() const
 			bboxArr.push_back({ bbox[1].x, bbox[1].y, bbox[1].z });
 		}
 
-		auto newMesh = repo::core::model::RepoBSONFactory::makeMeshNode(newVertices, newFaces, newNormals, bboxArr, newUVs, newColors);
+		auto submeshIds = getIDMapArrays();
```

Comment:
Not something to address here, just noting down my thoughts.

Stash graph and this should probably merge into 1 in the future? We're effectively batching the meshes twice 😆